### ISSUE TITLE
the model used to report baseline results

### ISSUE THREAD

**Issue by gxjlhx** [epic-kitchens/C10-epic-sounds-detection#2]
"The link to download the model used to report baseline results is broken."

**Comment by JacobChalk** [comment]
Hi, 

Would you be able to reference which link is broken? 

The link for the features: [here](https://www.dropbox.com/scl/fi/eo7fis5zxgaein0r130c2/auditory_slowfast_features.zip?rlkey=ytcb73it8fqatsrzainwda989&e=1&dl=0) and for the pretrained model [here](https://www.dropbox.com/scl/fi/uszgvn6xz7l543ald1n0p/EPIC_Sounds_ActionFormer.tar?rlkey=mk08iyfzr1wn5dxs3hwxy82m1&e=1&dl=0), appear to be working on our end. Equally, the baseline results on the validation set can be found [here](https://github.com/epic-kitchens/C10-epic-sounds-detection/blob/main/EvaluationCode/actionformer_baseline_validation.json).
